src/foundry/client/pixi/layers/placeables/tokens.d.ts

```diff
@@ -53,28 +57,26 @@ declare global {
      */
     get ownedTokens(): ReturnType<this["placeables"]["filter"]>;
 
-    override tearDown(): Promise<this>;
-
-    override activate(): this;
-
-    override deactivate(): this;
+    override _draw(options?: Record<string, unknown> | undefined): Promise<void>;
 
     /**
-     * @param options - (default: `{}`)
+     *
+     * @param options - Unused
```

Comment:
Not gonna note everywhere but this is another case to leave off if it's not put that way in Foundry.